### PR TITLE
Fix foodcritic 2x

### PR DESCRIPTION
**Pull request by bbaugher**
Test Kitchen Evidence: https://gist.github.com/bbaugher/a52e4d65317f97be2a93ec868e11f4ef

Same as #46 but also hit [FC074](http://www.foodcritic.io/#FC074) to use `default_action` method on our resource

### CODE REVIEW

**Comment by stefanmendoza** [comment]
+1

**Comment by mkwhitacre** [comment]
+1

**Comment by bbaugher** [comment]
Have enough +1s, merging

File: metadata.rb [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
 maintainer_email 'Bryan.Baugher@Cerner.com'
-license 'All rights reserved'
+license 'Apache-2.0'
 description 'Installs/Configures tomcat'
 long_description 'This cookbook is meant to install and configure an instance of tomcat'
 issues_url 'https://github.com/cerner/cerner_tomcat/issues'
 source_url 'https://github.com/cerner/cerner_tomcat'
 name 'cerner_tomcat'
 
+chef_version '>= 12.0' if respond_to?(:chef_version)
```
Comment (bjtieman) [inline metadata.rb:10]: What's the purpose of the respond_to? Why can't we just specify ">= 12.0"? And actually, given the problems we've encountered recently with Chef 13 should we just have "~ 12.0" for now?

Comment (bbaugher) [reply metadata.rb:10]: the purpose of respond_to is to handle situations where the version of chef doesn't have a `chef_version` method for metadata.rb which looks to have been [added in 12.6](https://docs.chef.io/config_rb_metadata.html). Currently this cookbook and its dependencies have no problems with Chef 13, our test kitchen runs still work fine so I left it open so we don't have to tweak and release it later

Comment (bjtieman) [reply metadata.rb:10]: +1...thanks!

**Pull request merged**